server/requestFormat.js

```diff
@@ -0,0 +1,57 @@
+testProductsArr = [
+	{	
+		productId: 12345,
+		productName: "Enkis fabulous product",
+		price: 10,
+		vendorId: 1,
+		vendorName: 'Enki',
+		quantity: 10,
+		isPrimeProduct: true
+	},
+	{	
+		productId: 12346,
+		productName: "some product",
+		price: 5,
+		vendorId: 2,
+		vendorName: 'Someone else',
+		quantity: 1,
+		isPrimeProduct: false
+	}
+]
+
+// Object that will be sent to transactions upon purchase:
+module.exports.processTransTestInput = {
+	userId: 123,
+	fullName: 'Name',
+  phone: '978-786-4567',
+	products: testProductsArr,
+	paymentId: 54321,
+	cartTotal: 105,
+	primeTrialSignUp: true,
+	shippingAddress: {
+    addressLine1: 'address1',
+    addressLine2: 'address2',
+    city: 'somecity',
+    state: 'CA',
+    zip: 12345,
+    country: 'USA'
+  },
+  billingAddress: {
+    addressLine1: 'address1',
+    addressLine2: 'address2',
+    city: 'somecity',
+    state: 'CA',
+    zip: 12345,
+    country: 'USA'
+  }
+}
+
+// Object that will be sent to transactions upon subscription:
+module.exports.subscribeTestInput = {
+  userId: 123, // this is a string of a userId
```

Comment:
The reason why the userId is a string is because Tim's Client service will send you an UUID, correct?